pkg/graph/resolvers/activity.resolvers.go

```diff
@@ -66,6 +66,23 @@ func (r *newModelPlanActivityMetaResolver) ModelPlan(ctx context.Context, obj *m
 	return ModelPlanGetByIDLOADER(ctx, obj.ModelPlanID)
 }
 
+// DataExchangeApproach is the resolver for the dataExchangeApproach field.
+func (r *planDataExchangeApproachMarkedCompleteActivityMetaResolver) DataExchangeApproach(ctx context.Context, obj *models.PlanDataExchangeApproachMarkedCompleteActivityMeta) (*models.PlanDataExchangeApproach, error) {
+	logger := appcontext.ZLogger(ctx)
+	//TODO: this can be replaced with a a call to // PlanDataExchangeApproachGetByModelPlanIDLoader() when model plan relation is added to the meta type
```

Comment:
It looks like we could call the data loader here instead of the get by ID method.

Reply:
8f5345d53